deploy/helm/values.dev.yaml

```diff
@@ -18,6 +18,8 @@ worker:
   image: acanthamoeba/sds-worker
   queueMode: azure
   queueEndpoint: https://<storageaccount>.queue.core.windows.net/runs
+  resourceGroup: resourceGroup
+  storageAccount: storageAccount
```

Comment:
Can we pick a more descriptive name? At a glance, it's confusing why this is "storageAccount", but then queueEndpoint is also "storageaccount". We don't want people to pick the same one here - they have distinct uses & access patterns. `runsTempStorageAccount` or something seems like a better fit

Reply:
That works. Just curious though, why would we not be able to use the same storage account. We'll just be creating new file shares and then deleting them post run so seemed like it wouldn't be too much of an issue using the same one

Reply:
The queue storage acct is network dual-homed - it's allowed to be accessed inside the detonation chamber for the worker, but also outside the detonation chamber by the laboratory. This temp share is purely internal and sometimes holds a copy of the data we're trying to protect during processing. We don't want that exposed outside in any way